src/core_plugins/kibana/public/management/react/index-pattern/index-pattern-create/index-pattern-create.component.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  KuiPageContent,
+  KuiPageContentBody,
+  KuiPageContentHeader,
+  KuiPageContentHeaderSection,
+  KuiTitle,
+  KuiText,
+  KuiForm,
+  KuiFormRow,
+  KuiSwitch,
+  KuiButton,
+  KuiFlexGroup,
+  KuiFlexItem,
+} from 'ui_framework/components';
+
+import { IndexPatternSearch } from './components/index-pattern-search';
+import { IndexPatternTimeFields } from './components/index-pattern-time-fields';
+import { IndexPatternResults } from './components/index-pattern-results';
+
+const IndexPatternCreate = ({
+  hasExactMatches,
+  isCreating,
+  isIncludingSystemIndices,
+  includeSystemIndices,
+  excludeSystemIndices,
+  createIndexPattern,
+}) => {
+  return (
+    <KuiPageContent>
+      <KuiPageContentHeader>
+        <KuiPageContentHeaderSection>
+          <KuiTitle>
+            <h2>Create new Index Pattern</h2>
+          </KuiTitle>
+          <KuiText size="small">
+            <p>
+              Index patterns are used to identify the Elasticsearch indices to run search and analytics against.
+              <br/>
+              They are also used to configure fields.
+            </p>
+          </KuiText>
+        </KuiPageContentHeaderSection>
+        <KuiPageContentHeaderSection>
+          <KuiSwitch
+            label="Include system indices"
+            checked={isIncludingSystemIndices}
+            onChange={() => isIncludingSystemIndices ? excludeSystemIndices() : includeSystemIndices()}
+          />
+        </KuiPageContentHeaderSection>
+      </KuiPageContentHeader>
+      <KuiPageContentBody>
+        <KuiFlexGroup>
+          <KuiFlexItem>
+            <KuiTitle>
+              <h3>Define a pattern...</h3>
```

Comment:
![2017-10-12 16_50_05](https://user-images.githubusercontent.com/5259/31502476-750722ec-af6d-11e7-8861-c6099314d03c.gif)

I hit some inconsistencies with handling of `*`

(just mentioning it in case it causes some architectural changes — haven't looked at the code for it)